checks/system/win32.py

```diff
@@ -114,10 +114,10 @@ def __init__(self, logger):
         self.gauge('system.mem.usable')
         self.gauge('system.mem.pct_usable')
         #  details about the usage of the pagefile.
-        self.gauge('system.mem.page_total')
-        self.gauge('system.mem.page_used')
-        self.gauge('system.mem.page_free')
-        self.gauge('system.mem.page_pct_free')
+        self.gauge('system.mem.pagefile_total')
```

Comment:
Why not `system.pagefile.total` ? (same for the others)
I know this has already been discussed, but why not `system.swap` ? Or maybe just an alias (two metric names for the same metric).